### ISSUE TITLE
Question about Action Downsampling

### ISSUE THREAD

**Issue by betray12138** [Robbyant/lingbot-va#44]
Congratulations on your great work !!

I notice that you have down-sampled the videos to 5-15fps for latent extraction.

I just feel curious about why you choose the scope of 5-15fps? 
Is it a direct consequence of 4x down-sampling from the pretraining datasets spannng from 20fps to 60 fps or some other reasons ？ 
and Why do not down-sample the action to align with 5-15fps as there may exist large discrepancy between the fps of actions among all the pre-training dataset? @lilin-hitcrt 

Thanks for your response in advance.

Cheers!

**Comment by lilin-hitcrt** [comment]
Thank you for your attention to our work. The selection of video and action frame rates is indeed an interesting topic.

1. If the video frame rate is excessively high, it will lead to a significant increase in computational load, thereby reducing the inference speed.
2. A too-low video frame rate can negatively impact model convergence and the accuracy of actions.
3. Actions require a higher frequency to enhance control precision.

Therefore, we recommend downsampling the video frames appropriately based on the original frame rate of your data. As for the specific frame rate to downsample to, this depends on your data characteristics—5-15 fps is just an empirical parameter from our experience.
